colcon_bundle/verb/utilities.py

```diff
@@ -252,5 +252,5 @@ def filechecksum(filename, algorithm='sha256', printing=False):
         if printing:
             print(filename + ' - ' + checksum)
         return checksum
-    except Exception as e:
+    except Exception as e: # noqa B902
```

Comment:
blind except was intended to be disabled from the get-go for good reason - any clue what types of exceptions we are expecting here - and is there any strong reason to always transform them into a `RuntimeError` as opposed to something more specific and informative?